### PR TITLE
Allow sorting users list by username.

### LINKED ISSUE #1522: Sort User List by Username
Allow the [list of users](https://dmoj.ca/users/) to be sorted by their usernames.

### PR DESCRIPTION
**Pull request by ComputerGenius152**
Closes #1522.

### CODE REVIEW

**Comment by ComputerGenius152** [comment]
CSS for the change is covered [here](https://github.com/DMOJ/online-judge/blob/88c1aa319250444230589ade04aef2e8bd68a6b9/templates/user/list.html#L10).

**Review by Ninjaclasher — APPROVED** [review]

File: templates/user/base-users-table.html [hunk 1]
```diff
@@ -2,9 +2,12 @@
 <tr>
     <th class="header rank">{{ rank_header or _("Rank") }}</th>
     {% block after_rank_head %}{% endblock %}
-    <th class="header username">{{ _('Username') }}</th>
+    <th class="header username">
+        {% if sort_links %}<a href="{{ sort_links.user__username }}">{% endif %}
```
Comment (kiritofeng) [inline templates/user/base-users-table.html:6]: Will this cause colours to change unexpectedly?

Comment (ComputerGenius152) [reply templates/user/base-users-table.html:6]: It doesn't seem to when I test it.
image

**Comment by kiritofeng** [comment]
Feature is a NACK, see discussion on #1522.

**Pull request closed without merging**